statick_tool/statick.py

```diff
@@ -358,6 +379,9 @@ def run(
                 plugin.scan(package, level, self.exceptions)
```

Comment:
the first discovery package run is going to seem to take much longer than all the rest because of how the [`find_files`](https://github.com/sscpac/statick/blob/main/statick_tool/discovery_plugin.py#L40) function works (it runs exactly once per package assuming the plugin was updated to support this structure, which most discovery plugins have done). i think it'd be worth adding separate timing for this function if it's not too difficult.

Reply:
[d31d427](https://github.com/sscpac/statick/pull/443/commits/d31d427f23c2e6ff69466ae62c83624f75cf54c1)

Good ideas, I think it looks much better now.